lib/ae_page_objects/elements/collection.rb

```diff
@@ -56,6 +56,11 @@ def configure(options)
       @item_locator = options.delete(:item_locator) || default_item_locator
     end
 
+    def options
+      evaled_locator = eval_locator(@item_locator)
```

Comment:
Yeah, I intentionally made it return a new object every time so that the result was safe to mutate.  There is no real performance gain to be had by memoizing here.